public/resources/remix/index.js

```diff
@@ -6,122 +6,126 @@
 
   var gaTrackingId = 'UA-68630113-1';
 
-  function injectAnalytics($) {
-    var analyticsHtml =
-    '\n<script>\n' +
-    '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' +
-    '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' +
-    'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' +
-    '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' +
-    'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' +
-    'ga(\'send\', \'pageview\');\n' +
-    '</script>\n';
-
-    $(head).append(analyticsHtml);
+  function injectAnalytics() {
+    var analytics = document.createElement("script");
+    analytics.textContent = '(function(i,s,o,g,r,a,m){i[\'GoogleAnalyticsObject\']=r;i[r]=i[r]||function(){\n' + '(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),\n' + 'm=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)\n' + '})(window,document,\'script\',\'//www.google-analytics.com/analytics.js\',\'ga\');\n' + 'ga(\'create\', \'' + gaTrackingId + '\', \'auto\');\n' + 'ga(\'send\', \'pageview\');\n';
+    head.appendChild(analytics);
   }
 
-  function setupBar(err, $) {
+  function setupBar(err) {
     if(err) {
       console.log("[Thimble Error] Unable to inject Remix UI. Error was: `" + err + "`");
       return;
     }
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
```

Comment:
Also, in the injectDetailsBar, we do -
`callback(textStatus)` when the return status of the xmlhttp request is bad. If I remove that line (because we're going to remove the callback function), should we have a console.log saying there was some error?